sigs.yaml

```diff
@@ -3111,22 +3111,22 @@ workinggroups:
   label: multitenancy
   leadership:
     chairs:
-    - github: srampal
-      name: Sanjeev Rampal
-      company: Cisco
-    - github: tashimi
+     - github: tashimi
       name: Tasha Drew
       company: VMware
     emeritus_leads:
     - github: davidopp
       name: David Oppenheimer
+    - github: srampal
```

Comment:
oh, would be good to get an ack on this move from @srampal or a reference to where this was communicated

Reply:
Communicated here: https://groups.google.com/g/kubernetes-wg-multitenancy/c/fX3ZnmKkDL0 

I asked @srampal to do this several months ago but he hasn't replied, he hasn't been part of the group for about two years now, so removing him and adding the new chairs is appropriate.